src/style.css

```diff
@@ -0,0 +1,379 @@
+  float: left;
+  line-height: 24px;
+  padding-top: 8px;
+  margin-left: 24px;
+}
+
+header nav svg {
+  height: 16px;
+}
+
+header nav svg.logo {
+  height: 24px;
+  position: relative;
+  top: 8px;
+}
+
+header nav .f-left svg {
+  margin-right: 16px;
+}
+
+header nav .f-left span {
+  font-weight: bold;
+}
+
+header nav .f-right {
+  float: right;
+  margin-right: 24px;
+}
+
+header nav button {
+  height: 28px;
+  background-color: #6288a5;
+  border: none;
+  color: #fff;
+  font-weight: bold;
+  font-size: 10px;
+  padding: 0 12px;
+  border-radius: 4px;
+}
+
+/* ASIDE */
+aside {
+  width: 270px;
+  float: right;
+  top: 960px;
+  margin-right: 12px;
+}
+
+aside > h2 {
+  font-size: 18px;
+  font-weight: bold;
+}
+
+aside > div {
+  height: 90px;
+  display: inline-flex;
+}
+
+aside > div img {
+  height: 70px;
+}
+
+aside > div .title {
+  font-size: 14px;
+  margin-left: 4px;
+  text-align: justify;
+  line-height: 18px;
+}
+
+/* ARTICLE / MAIN  */
+
+article {
+  width: 1200px;
+  margin: 80px auto;
+}
+
+article a {
+  color: #326891;
+}
+
+.article-title {
+  float: left;
+  margin-left: 260px;
+  font-family: "Merriweather", serif;
+  font-size: 1.5rem;
+  font-style: italic;
+  font-weight: 700;
+  width: 450px;
+}
+
+article .title-img {
+  width: 900px;
+  margin: 0 auto;
+}
+
+article .title-img > img {
+  width: 100%;
+}
+
+.image-desc {
+  width: 850px;
+  float: left;
+  color: rgb(63, 63, 63);
+}
+
+.credit {
+  font-size: 14px;
+}
+
+.author-date-wrapper {
+  float: left;
+}
+
+.article-info,
+.article {
+  margin: 0 auto;
+  width: 600px;
+}
+
+article .social-media {
+  float: right;
+}
+
+article .social-media ul li {
+  list-style-type: none;
+  display: inline;
+}
+
+article .social-media img {
+  width: 15px;
+  padding: 8px;
+}
+
+.article {
+  float: left;
+  font-size: 21px;
+  line-height: 27px;
+  margin-left: 300px;
+}
+
+.img-caption {
+  font-size: 16px;
+  line-height: 19px;
+  color: rgb(63, 63, 63);
+}
+
+.collapse-parent {
+  position: relative;
+  width: 900px;
+  margin-left: -140px;
+}
+
+#diagram-img {
+  width: inherit;
+}
+
+#telescope-img {
+  width: 100%;
+}
+
+.diagram-img-text {
+  position: absolute;
+  width: 110%;
+  left: 0;
+  top: 130px;
+}
+
+.diagram-img-caption {
+  display: inline-block;
+  width: 204px;
+  height: 100px;
+  margin: 35px 10px;
+  font-family: "Raleway", sans-serif;
+  padding-bottom: 90px;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.bold-text {
+  font-family: "Raleway", sans-serif;
+  text-transform: uppercase;
+  font-weight: 700;
+}
+
+/* BOTTOM SECTION */
+section.bottom-sec {
+  width: calc(100% - 160px);
+  margin: 0 80px;
+}
+
+section.bottom-sec .section-title {
+  font-size: 24px;
+  font-weight: bold;
+  margin: 24px 0;
+}
+
+section.bottom-sec > .more-in-category {
+  width: 66%;
+  float: left;
+  border-top: 1px solid black;
+  margin: 40px 0;
+}
+
+section.bottom-sec > .more-in-category .article-thumb {
+  display: inline-block;
+  width: 33%;
```

Comment:
Let's reduce this to `30%`.

Currently, the **More in Space & Cosmos** section is messed up with a zoom percentage increase. I initially viewed it with `100%` zoom and notice this. Changing to `30%` will reduce the unnecessary space this element is taking and consequently allowing three elements to be in a single row. See below:

![nyt-misnc](https://user-images.githubusercontent.com/36057474/65366340-74e5fa00-dc1a-11e9-82f7-0371b5b8fec3.gif)

Reply:
Resolved.